src/snap/metric.cc

```diff
@@ -100,10 +113,10 @@ const std::vector<Metric::NamespaceElement>& Metric::ns() const {
 
 std::vector<int> Metric::dynamic_ns_elements() const {
```

Comment:
It seems that this method duplicates `Namespace::get_dynamic_indexes` . Am I right? 
If I'm right please remove this function.